airbyte-integrations/connectors/source-relational-db/src/main/java/io/airbyte/integrations/source/relationaldb/AbstractDbSource.java

```diff
@@ -77,7 +78,6 @@
   // TODO: Remove when the flag is not use anymore
   private final FeatureFlags featureFlags = new EnvVariableFeatureFlags();
 
-  @Override
```

Comment:
I prefer to have the Override keyword! Lets keep these Override keywords cause they help avoid issues by throwing compilation error in case the super class behaviour changes.

Reply:
Added the override back